app/layout/projects/list/component.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import cx from 'classnames';
+
+import { useSelector } from 'react-redux';
+
+import Wrapper from 'layout/wrapper';
+import Item from 'components/projects/item';
+
+import { useProjects } from 'hooks/projects';
+
+export interface ProjectsListProps {
+
+}
+
+export const ProjectsList: React.FC<ProjectsListProps> = () => {
+  const { search } = useSelector((state) => state['/projects']);
```

Comment:
@mbarrenechea why did you name the root pointer `/projects` instead of just `projects`. Is it a new code convention from newer Redux versions?

Reply:
It's just an idea, nothing related to conventions